coralproject/talk
# Add missing indices for `dsaReports` to `INDEXES.md`

## What does this PR do?

Add missing indices for `dsaReports` to `INDEXES.md`

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

```
db.dsaReports.createIndex({ tenantID: 1, id: 1 }, { unique: true });
db.dsaReports.createIndex({ status: 1, createdAt: 1, tenantID: 1 });
db.dsaReports.createIndex({ referenceID: 1, tenantID: 1 });
db.dsaReports.createIndex({ submissionID: 1, tenantID: 1 });
```

Added to `INDEXES.md`.

## How do I test this PR?

- Use mongo compass to create indices
- Use DSA functionality, see usage of the indices for their specific purposes

## Where any tests migrated to React Testing Library?

No

## How do we deploy this PR?

- When DSA is deployed to an instance, add these indexes to their appropriate collections.
